Makefile

```diff
@@ -134,7 +134,7 @@ LIB_LDLIBS_STATIC  += -l:libtirpc.a
 LIB_LDLIBS_SHARED  += -lpthread
 endif
 ifeq ($(WITH_SECCOMP), yes)
-LIB_CPPFLAGS       += -DWITH_SECCOMP
+LIB_CPPFLAGS       += -DWITH_SECCOMP $(shell pkg-config --silence-errors --cflags libseccomp)
```

Comment:
--silence-errors ? Do we really want to do that ? Shouldn't we fail here instead of later at link-time ?

Reply:
Agreed. Unfortunately even without `--silence-errors` will `make` continue, but at least there will be a visible error message.